doc/code/SqlToSlick.scala

```diff
@@ -4,421 +4,425 @@ import scala.concurrent.Await
             from PERSON
             group by ADDRESS_ID
-          """.as[(Int,Option[Int])]
-          //#sqlQueryGroupBy
+          """.as[(Int, Option[Int])]
+        // #sqlQueryGroupBy
         val slick =
-          //#slickQueryGroupBy
-          people.groupBy(p => p.addressId)
-                 .map{ case (addressId, group) => (addressId, group.map(_.age).avg) }
-                 .result
-          //#slickQueryGroupBy
+          // #slickQueryGroupBy
+          people.groupBy(p => p.addressId).map { case (addressId, group) => (addressId, group.map(_.age).avg) }.result
+        // #slickQueryGroupBy
         val (sqlRes, slickRes) = Await.result(db.run(sql zip slick), Duration.Inf)
         assert(sqlRes == slickRes)
         assert(sqlRes.size > 0)
         assert(sqlRes.exists(_._2 == Some(1000)))
-      };{
+      }
+      {
         val sql =
-          //#sqlQueryHaving
+          // #sqlQueryHaving
           sql"""
             select ADDRESS_ID
             from PERSON
             group by ADDRESS_ID
             having avg(AGE) > 50
           """.as[Int]
-          //#sqlQueryHaving
+        // #sqlQueryHaving
         val slick =
-          //#slickQueryHaving
-          people.groupBy(p => p.addressId)
-                 .map{ case (addressId, group) => (addressId, group.map(_.age).avg) }
-                 .filter{ case (addressId, avgAge) => avgAge > 50 }
-                 .map(_._1)
-                 .result
-          //#slickQueryHaving
+          // #slickQueryHaving
+          people
+            .groupBy(p => p.addressId)
+            .map { case (addressId, group) => (addressId, group.map(_.age).avg) }
+            .filter { case (addressId, avgAge) =>
+              avgAge > 50
+            }
+            .map(_._1)
+            .result
+        // #slickQueryHaving
         val (sqlRes, slickRes) = Await.result(db.run(sql zip slick), Duration.Inf)
         assert(sqlRes == slickRes)
         assert(sqlRes.size > 0)
-      };{
+      }
+      {
         val sql =
-          //#sqlQueryImplicitJoin
+          // #sqlQueryImplicitJoin
           sql"""
             select P.NAME, A.CITY
             from PERSON P, ADDRESS A
             where P.ADDRESS_ID = a.id
-          """.as[(String,String)]
-          //#sqlQueryImplicitJoin
+          """.as[(String, String)]
+        // #sqlQueryImplicitJoin
         val slick =
-          //#slickQueryImplicitJoin
-          people.flatMap(p =>
-            addresses.filter(a => p.addressId === a.id)
-                     .map(a => (p.name, a.city))
-          ).result
+          // #slickQueryImplicitJoin
+          people.flatMap(p => addresses.filter(a => p.addressId === a.id).map(a => (p.name, a.city))).result
 
-          //#slickQueryImplicitJoin
+        // #slickQueryImplicitJoin
         val slick2 =
-          //#slickQueryImplicitJoin
+          // #slickQueryImplicitJoin
           // or equivalent for-expression:
-          (for(p <- people;
```

Comment:
_something_ removed the semicolon here. i can't find anywhere in scalafmt where that would happen.

Reply:
This was likely me doing it manually since there were other places with redundant semicolons and I likely made a mistake here

Reply:
perhaps you could enable the `PreferCurlyFors` rules, and it would have done it for you. along with replacing `()` with `{}`, of course.